sql/template.sql

```diff
@@ -88,6 +88,7 @@ CREATE TABLE IF NOT EXISTS organization.classification_sets (
   created_by_resource_id text,
   updated_by_id bigint,
   updated_by_resource_id text,
+  system_type text NOT NULL,
```

Comment:
Are you sure this can't be null???? Seems wrong to me.

Reply:
looking at other system_type columns they seem to be nullable as well